test/9.custom-request-clients.test.js

```diff
@@ -0,0 +1,119 @@
+/* global describe, it */
+'use strict'
+
+const request = require('supertest')
```

Comment:
Thankfully [`supertest` sets `.redirects(0)`](https://github.com/visionmedia/supertest/blob/ed9d49fd366472282f644479f3de9b9d82a8a6df/lib/test.js#L32) so my testing of whether custom clients can follow redirects or not is valid.

Reply:
Great!